cmd/controller/certificates.go

```diff
@@ -66,7 +68,7 @@ func (c *Certificates) Init(ctx context.Context) error {
 	c.CACert = string(cert)
 	// Changing the URL here also requires changes in the "k0s kubeconfig admin" subcommand.
 	apiAddress := net.JoinHostPort(c.ClusterSpec.API.Address, strconv.Itoa(c.ClusterSpec.API.Port))
-	kubeConfigAPIUrl := fmt.Sprintf("https://%s", apiAddress)
+	kubeConfigAPIUrl := (&url.URL{Scheme: "https", Host: apiAddress}).String()
```

Comment:
There's probably a constant for `https` that the `usestdlib` linter will complain about :)

Reply:
Funny enough, there isn't!